src/main/java/ToDo.java

```diff
@@ -0,0 +1,11 @@
+public class ToDo extends Task {
```

Comment:
I notice that ToDo doesn't override the getStatusIcon method from Task, whereas Deadline and Event does. Is this intended? And also since Task will never be called in your program, would it be better to just delete the overriding methods and just make the change in the method in the Task class?

Reply:
Oh yes this was intended, because I felt like for deadlines and events, they shouldn't print a cross when not completed, only a tick when completed (because there is a time aspect for those).